techniques/failures/F99.html

```diff
@@ -2,133 +2,78 @@
-                  active when a particular interface element has focus. For example, when a select element
-                  or a custom listbox has focus, the input of single character keys to navigate the
-                  list is a useful feature.
-               </p>
+               <div role="heading" class="note-title marker" aria-level="3">注記</div>
+               <p>単一キーのキーボードショートカットの使用は、特定のインターフェース要素がフォーカス持つ場合のみショートカットがアクティブであるときは失敗ではない。例えば、セレクト要素又はカスタムリストボックスがフォーカスを持つ場合、リストをナビゲートするための単一の文字キーの入力は便利な機能である。</p>
             </div>
-            <p>The test procedure suggests asking the author (often the developer of the site) whether
-               keyboard shortcuts are used. If that information is trusted then the procedure can
-               be simpler than pressing all the keys.
-            </p>
-            <p>The success criterion does not apply when single key shortcuts are only active when
-               interface elements have the focus, for example, a <code>select</code> element. Here, pressing a letter key is used for fast navigation within the select
-               options.
-            </p>
-            <p>Viewing page scripts and searching for typical keyboard event handlers like <code>document.addEventListener('keydown' ...) </code> or the presence of the <code>.keycode</code> attribute
-               may establish the presence of scripts that intercept keyboard shortcuts without modification
-               keys like ALT or
-               STR being held down at the same time. As there are several ways of implementing character
-               key events, this
-               method is not considered reliable.
-            </p>
-            <p>Some browsers employ single key shortcuts with <kbd>Shift</kbd>. For example, Firefox opens a page search when pressing
-               <kbd>Shift</kbd> + <kbd>/</kbd> and a search in page links when pressing <kbd>Shift</kbd> + <kbd>'</kbd>. In these cases, it will be necessary to press <kbd>Esc</kbd> or click an empty part of the page to remove the focus from the browser input.
-            </p>
+            <p>この試験方法はキーボードショートカットが使用されているかどうかを制作者 (しばしばサイトの開発者) へ尋ねることを提案している。 その情報が信頼に足る場合、すべてのキーを押下するより手順は簡略化するはずである。</p>
+            <p>例えば、<code>select</code> 要素のようなインタフェース要素がフォーカスを持つ場合、単一キーのショートカットが有効なとき、達成基準は適用されない。この場合、アルファベットキーの押下が、選択オプション内での高速なナビゲーションのために利用される。</p>
+            <p>ページスクリプトの表示、及び <code>document.addEventListener('keydown' ...)</code> のような典型的なキーボードイベントハンドラ又は <code>.keycode</code> 属性の有無の検索により、Alt 又は Str のような同時に押下される装飾キーなしにキーボードショートカットを妨害するスクリプトの存在を証明してもよい。文字キーのイベントを実装する方法は複数あるため、この方法は信頼性が低いと考えられている。</p>
+            <p>いくつかのブラウザは <kbd>Shift</kbd> を併用した単一キーショートカットを採用している。例えば、Firefox は <kbd>Shift</kbd> + <kbd>/</kbd> を押下した場合はページ検索を開く、及び <kbd>Shift</kbd> + <kbd>'</kbd> を押下した場合はページリンクを検索する。いずれの場合でも、ブラウザの入力からフォーカスを削除するためには、Esc を押下する又はページの空である部分をクリックする必要があるはずである。</p>
```

Comment:
例示の中なので「及び」にしない方向で
removeは「削除」ではない（deleteではない）と思うので

```suggestion
            <p>いくつかのブラウザは <kbd>Shift</kbd> を併用した単一キーショートカットを採用している。例えば、Firefox は <kbd>Shift</kbd> + <kbd>/</kbd> を押下した場合はページ検索を開き、<kbd>Shift</kbd> + <kbd>'</kbd> を押下した場合はページリンクを検索する。いずれの場合でも、ブラウザの入力からフォーカスを外すためには、Esc を押下する又はページの空である部分をクリックする必要があるはずである。</p>
```